examples/demo-rollup/README.md

```diff
@@ -233,51 +236,53 @@ Arguments:
+$ cat ./examples/demo-stf/src/sov-cli/test_data/create_token.dat
 7cb06da843cb98a223cdd4aee61ea4533f99104fe03144720d75800580d9a665be112c73b8d0b02b8de73f678d2432e93f613071e6fd04cc96b6ab5e6952bf007b758bf2e7670fafaf6bf0015ce0ff5aa802306fc7e3f45762853ffc37180fe66800000001000b000000000000000e000000736f762d746573742d746f6b656ee803000000000000a3201954f70ad62230dc3d840a5bf767702c04869e85ab3eee0b962857ba759801000000a3201954f70ad62230dc3d840a5bf767702c04869e85ab3eee0b962857ba75980100000000000000
 ```
-* The above is the hex representation of the serialized transaction
-* The transaction is however not yet ready to be submitted to celestia, since celestia accepts blobs which can contain multiple transactions
-* There is another subcommand for `sov-cli` that can bundle serialized transaction files into a blob
+The above is the hex representation of the serialized transaction. There is another subcommand for `sov-cli` that can bundle serialized transaction files into a blob:
 ```
-user@machine sovereign % ./target/debug/sov-cli make-blob -h
+$ ./target/debug/sov-cli make-blob -h
 Usage: sov-cli make-blob [PATH_LIST]...
 
 Arguments:
   [PATH_LIST]...  List of serialized transactions
 ```
-* We have only one transaction, so we'll use that to create the serialized file
+We only have one transaction, so we can use that transaction to create the serialized blob, using the `make-blob` `sov-cli` subcommand:
 ```
-user@machine sovereign % ./target/debug/sov-cli make-blob ./examples/demo-stf/src/sov-cli/test_data/create_token.dat 
+$ ./target/debug/sov-cli make-blob ./examples/demo-stf/src/sov-cli/test_data/create_token.dat 
 01000000d40000007cb06da843cb98a223cdd4aee61ea4533f99104fe03144720d75800580d9a665be112c73b8d0b02b8de73f678d2432e93f613071e6fd04cc96b6ab5e6952bf007b758bf2e7670fafaf6bf0015ce0ff5aa802306fc7e3f45762853ffc37180fe66800000001000b000000000000000e000000736f762d746573742d746f6b656ee803000000000000a3201954f70ad62230dc3d840a5bf767702c04869e85ab3eee0b962857ba759801000000a3201954f70ad62230dc3d840a5bf767702c04869e85ab3eee0b962857ba75980100000000000000
 ```
-* The output can be redirected to a file so that we can use it with the `make` command from earlier
+4. Lets create the serialized blob and redirect the output to a file so that we can use it later:
 ```
-user@machine sovereign % ./target/debug/sov-cli make-blob ./examples/demo-stf/src/sov-cli/test_data/create_token.dat > ./examples/demo-stf/src/sov-cli/test_data/celestia_blob
+$ ./target/debug/sov-cli make-blob ./examples/demo-stf/src/sov-cli/test_data/create_token.dat > ./examples/demo-stf/src/sov-cli/test_data/celestia_blob
 ```
-* To submit the blob, we'll start from scratch (since the test transaction we submitted has the same nonce, token fields etc)
+
+5. To submit the blob, we'll start the DA layer instance and the rollup from scratch since the test transaction we submitted (as part of the sanity check) has the same nonce, token fields etc.:
```

Comment:
```suggestion
5. To submit the blob, let's start the DA layer instance and the rollup from scratch (since the test transaction we submitted as part of the sanity check has the same nonce, and would cause an error as a result) :
```